crates/swc/tests/tsc-references/bitwiseNotOperatorWithStringType.1.normal.js

```diff
@@ -22,7 +22,15 @@ var A = /*#__PURE__*/ function() {
 var M;
 (function(M) {
     var n;
-    M.n = n;
+    Object.defineProperty(M, "n", {
```

Comment:
Is this change intentional?

Reply:
It looks like a regression of the output size

Reply:
Yes, `export let/var` in TypeScript means mutable exported bindings.

Reply:
This change is a bugfix of #5259.
If output size is more important, I can change it to have the same output as `export const`.

Reply:
I think it's better to optimize for the output size because `isolatedModules: true` is our assumption, and `tsc` works similar under same assumptions

Reply:
OK, I will have a anoter try.

Reply:
@kdy1 done.
I will try to fix #5259 in another pull request.